pyproject.toml

```diff
@@ -104,10 +104,6 @@ requires = ["poetry-core"]
 build-backend = "poetry.core.masonry.api"
 
 [tool.pyright]
-include = [
-  "src/backend/services/metrics.py", 
-  "src/backend/tools/google_drive/sync/actions/",
-  ]
 defineConstant = { DEBUG = true }
```

Comment:
Do we need to check all the code? If so, I think we should set it to ./src/backend to check only the Python code.

Reply:
Let me try

Reply:
I had to revert this removal, even with the updated pyright checker it checks types across the entire backend - we have 900+ errors so there's no question that we don't have bandwidth to resolve them, I'll revert to this for now (which seems to resolve the error)